charts/rancher-gatekeeper-operator/v3.1.0-beta.700/values.yaml

```diff
@@ -6,7 +6,7 @@ image:
   repository: quay.io/open-policy-agent/gatekeeper
```

Comment:
Wasn't quite sure but looks like we have to a pattern to use images under `rancher` repo. Can we add this to our mirror scripts https://github.com/rancher/rancher/blob/master/scripts/istio-images-mirror?

Reply:
Yes if we want to do airgap, in order to get them included in rancher-images.txt (which people use as a list of images to download) then all images must be in format of:
```
repository: user/image
tag: 1.2.3
```
You can test this from rancher by running `go run main.go ~/path/to/system-charts` from rancher/pkg/image/export/main.go. You'll see that this one wasn't picked up, or at least its tag wasn't. The istio-images-mirror script is what gets them into our dockerhub.

Reply:
Added to the script https://github.com/rancher/rancher/pull/25578